intro/intro.rst

```diff
@@ -300,49 +420,46 @@ Other useful magic functions are:
 
   .. sourcecode:: ipython
 
-    In [2]: cd /tmp
+    In [1]: cd /tmp
     /tmp
 
-* ``%timeit`` allows you to time the execution of short snippets using the
-  ``timeit`` module from the standard library:
-
-  .. sourcecode:: ipython
-
-      In [3]: timeit x = 10
-      10000000 loops, best of 3: 39 ns per loop
-
 * ``%cpaste`` allows you to paste code, especially code from websites which has
   been prefixed with the standard Python prompt (e.g. ``>>>``) or with an ipython
   prompt, (e.g. ``in [3]``):
 
   .. sourcecode:: ipython
 
-    In [5]: cpaste
-    Pasting code; enter '--' alone on the line to stop or use Ctrl-D.
-    :In [3]: timeit x = 10
-    :--
-    10000000 loops, best of 3: 85.9 ns per loop
-    In [6]: cpaste
+    In [2]: %cpaste
     Pasting code; enter '--' alone on the line to stop or use Ctrl-D.
-    :>>> timeit x = 10
+    :>>> for i in range(3):
+    :...     print(i)
     :--
-    10000000 loops, best of 3: 86 ns per loop
+    0
+    1
+    2
+
+* ``%timeit`` allows you to time the execution of short snippets using the
+  ``timeit`` module from the standard library:
+
```

Comment:
see also to mention the section on profiling? http://www.scipy-lectures.org/advanced/optimizing/index.html#id4